dist-material/bin/webappServiceShutdown.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PRG="$0"
+PRGDIR=$(dirname "$PRG")
+[ -z "$SW_HOME" ] && SW_HOME=$(cd "$PRGDIR/.." > /dev/null || exit 1; pwd)
+
+UI_PID_FILE="${SW_HOME}/bin/ui.pid"
+
+if [ -f $UI_PID_FILE ]; then
+  kill -9 $(cat "$UI_PID_FILE")
```

Comment:
Do we really need `-9`?

Reply:
After careful consideration, the current scenario does not need kill - 9, and using the kill command can meet the needs of graceful downtime

Reply:
@kezhenxu94 I think with using `graceful shutting down`, we need a simple e2e to verify whether OAP and UI could be shut down successfully. Because, once we have one unexpected thread, the shutdown wouldn't work.

Reply:
The simplest version is to update `JDK-version` CI process to add `shutdown` and `check` at the end of processes. @kezhenxu94 Make sense?

Reply:
> The simplest version is to update `JDK-version` CI process to add `shutdown` and `check` at the end of processes. @kezhenxu94 Make sense?

Track here https://github.com/apache/skywalking/issues/7022